src/NzbDrone.Core/Notifications/Twitter/TwitterService.cs

```diff
@@ -83,9 +83,19 @@ public void SendNotification(string message, TwitterSettings settings)
 
                 if (settings.DirectMessage)
                 {
-                    twitter.DirectMessage(message, settings.Mention);
-                }
-
+                    if(settings.Mention.Contains(","))
+					{
+						var recipients = settings.Mention.Replace(" ",string.Empty).Split(',');
```

Comment:
Using `StringSplitOptions.RemoveEmptyEntries` would avoid trying to send an invalid mention.

Better yet having spaces in the mention field should be a validation error so it doesn't save it, if we want a safety net we should use `Trim()` after the split, not just blindly remove spaces.